docs/pages/application-access/introduction.mdx

```diff
@@ -35,23 +35,31 @@ internal dashboards and applications, such as:
   Your browser does not support the video tag.
 </video>
 
-## Getting Started
+## Getting started
 
 Get started with Application Access in a 10 minute [guide](./getting-started.mdx).
 
 ## Guides
 
-- [Connecting Applications](./guides/connecting-apps.mdx)
-- [JWT Tokens](./guides/jwt.mdx)
-- [API Access](./guides/api-access.mdx)
+<TileSet>
+  <Tile icon="window" title="Connecting Applications" href="./guides/connecting-apps.mdx">
+    How to use Teleport for Application Access.
+  </Tile>
+  <Tile icon="window" title="JWT Tokens" href="./guides/jwt.mdx">
+    How to use JWT tokens with Application Access.
```

Comment:
Can we make this more explicit? 

How to use JWT tokens with Application Access to ...? ( automatically login to applications )